subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -170,6 +170,158 @@ config BT_TBS_CLIENT_MAX_TBS_INSTANCES
 	  Sets the maximum number of Telephone Bearer Service (TBS)
 	  instances to setup and use.
 
+config BT_TBS_CLIENT_BEARER_PROVIDER_NAME
+	bool "Read Bearer Provider Name Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Provider Name
+	  procedure.
+
+config BT_TBS_CLIENT_BEARER_UCI
+	bool "Read Bearer UCI Procedure"
+	default y
```

Comment:
We default everything to `y` it seems. Nothing wrong with that, but I do wonder if it should be the other way around?

If we keep it as `default y`, I would suggest to also create Kconfig along the lines of `BT_TBS_CLIENT_MINIMAL` that disables all optional features.